Assets/HoloToolkit/Input/Scripts/InteractableViaInterface.cs

```diff
@@ -0,0 +1,19 @@
+﻿using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    public class InteractableViaInterface : MonoBehaviour, IInteractable
```

Comment:
It might make more sense to convert the example/test scripts to use the interface or events.

Also, you could make the class SendMessage("OnSelect") in the OnTap() handler.  Lazy...err..busy people could attach the script to their objects when we deprecate OnSelect.